lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -1517,6 +1517,60 @@ void PrimUpdaterManager::discardPullSetIfEmpty()
     }
 }
 
+static void replicateMissingAncestors(
+    const UsdStageRefPtr& srcStage,
+    SdfPath               srcPath,
+    const UsdStageRefPtr& dstStage,
+    SdfPath               dstPath)
+{
+    // List f prims to be created. The last prim should be created
+    // first, (it will be the highest in the hierarchy)
+    std::vector<std::pair<SdfPath, SdfPath>> toBeCreated;
+
+    while (true) {
+        // If we reach the top of the hierarchy, stop.
+        srcPath = srcPath.GetParentPath();
+        if (srcPath.IsEmpty() || srcPath.IsAbsoluteRootPath())
+            break;
+
+        // If we reach the top of the hierarchy, stop.
+        dstPath = dstPath.GetParentPath();
+        if (dstPath.IsEmpty() || dstPath.IsAbsoluteRootPath())
+            break;
+
+        // If the destination prim already exists, stop.
+        UsdPrim dstPrim = dstStage->GetPrimAtPath(dstPath);
```

Comment:
Is it possible that the parent of this `dstPrim` needed to be created? I see that if `dstPrim` exists we just break the loop here.

Reply:
No, if the prim exist, its parent necessarily exists.